format_checker/projects.json

```diff
@@ -0,0 +1 @@
cts/spring-data-jdbc": "unforked", "https://github.com/spring-projects/spring-data-jpa": "unforked", "https://github.com/spring-projects/spring-data-keyvalue": "unforked", "https://github.com/spring-projects/spring-data-r2dbc": "unforked", "https://github.com/spring-projects/spring-data-rest": "unforked", "https://github.com/spring-projects/spring-hateoas": "unforked", "https://github.com/spring-projects/spring-security-oauth": "unforked", "https://github.com/spring-projects/spring-ws": "unforked", "https://github.com/square/burst": "unforked", "https://github.com/square/catalog-api-demo": "unforked", "https://github.com/square/dagger": "unforked", "https://github.com/square/otto": "unforked", "https://github.com/square/pagerduty-incidents": "unforked", "https://github.com/square/retrofit": "unforked", "https://github.com/stanfordnlp/CoreNLP": "unforked", "https://github.com/stargate/stargate": "unforked", "https://github.com/stleary/JSON-java": "unforked", "https://github.com/streamx-co/FluentJPA": "unforked", "https://github.com/takari/ollie": "unforked", "https://github.com/tbsalling/aismessages": "unforked", "https://github.com/Tencent/spring-cloud-tencent": "unforked", "https://github.com/TestingResearchIllinois/starts": "unforked", "https://github.com/Thomas-S-B/visualee": "unforked", "https://github.com/tomsun28/sureness": "unforked", "https://github.com/tools4j/unix4j": "unforked", "https://github.com/TooTallNate/Java-WebSocket": "unforked", "https://github.com/twilio/authy-java": "unforked", "https://github.com/twitter/cloudhopper-commons": "unforked", "https://github.com/twitter/GraphJet": "unforked", "https://github.com/twitter/hraven": "unforked", "https://github.com/undertow-io/undertow": "unforked", "https://github.com/vaadin/flow": "unforked", "https://github.com/vert-x3/vertx-mqtt": "unforked", "https://github.com/vert-x3/vertx-rabbitmq-client": "unforked", "https://github.com/vipshop/vjtools": "unforked", "https://github.com/visallo/vertexium": "unforked", "https://github.com/vmware/admiral": "unforked", "https://github.com/vojtechhabarta/typescript-generator": "unforked", "https://github.com/webx/citrus": "unforked", "https://github.com/wikidata/wikidata-toolkit": "unforked", "https://github.com/wikimedia/search-highlighter": "unforked", "https://github.com/wildfly/jboss-ejb-client": "unforked", "https://github.com/wildfly/wildfly": "unforked", "https://github.com/wildfly/wildfly-core": "unforked", "https://github.com/wildfly/wildfly-maven-plugin": "unforked", "https://github.com/winder/Universal-G-Code-Sender": "unforked", "https://github.com/worldline/tapestry-cdi": "forked", "https://github.com/wro4j/wro4j": "unforked", "https://github.com/wso2/balana": "unforked", "https://github.com/wso2/carbon-apimgt": "unforked", "https://github.com/yandex/yoctodb": "unforked", "https://github.com/yangfuhai/jboot": "unforked", "https://github.com/zalando/riptide": "unforked", "https://github.com/zhangxd1989/spring-boot-cloud": "unforked"}
```

Comment:
How did you generate this list? Did you use some script or compile it manually?

Reply:
I used a script. The `check_forked_project` will record the project if it is not in the JSON file. And the `forked_pr_remover.py` file used the function to filter the existing tests and generated this file.

Reply:
Using the script is fine, but for `pr-data.csv`, you need to only remove some lines not change all lines.

Reply:
Just resolved the format issue, using "\n" as the line terminator instead of the default "\r\n".

Reply:
Great work on removing forked repos. Can you add original repos (as per my email)?

Reply:
No problem.